base/array.jl

```diff
@@ -752,9 +837,45 @@ function findnext(A, v, start::Integer)
     end
     return 0
 end
+"""
+    findfirst(A, v)
+
+Return the linear index of the first element equal to `v` in `A`.
+Returns 0 if `v` is not found.
```

Comment:
Since the `0` being referred to is a possible return value of the function it should probably be formatted as code. If it was the actual number 0, rather than the Julia value, then a "raw" 0 would be better I'd say.